mne/viz/_3d.py

```diff
@@ -2943,7 +2943,8 @@ def snapshot_brain_montage(fig, montage, hide_sensors=True):
 
 
 @fill_doc
-def plot_sensors_connectivity(info, con, picks=None):
+def plot_sensors_connectivity(info, con, picks=None,
+                              cbar_label='connectivity'):
```

Comment:
```suggestion
                              cbar_label='Connectivity'):
```
Since it's a title maybe capitolize?